### ISSUE TITLE
"Form is being processed"

### ISSUE THREAD

**Issue by markendley** [SmackBang/tmbrmll#13]
Because it's such a huge form with so much info to send to the server, I had to make the "waiting" message more prominent.

Right now it needs a bit of design to make it more user friendly. Maybe an animated preloader icon, or just an icon of a stopclock or something.

You ladies will see what I mean when you try submitting the form.

![screen shot 2016-05-10 at 7 34 47 pm](https://cloud.githubusercontent.com/assets/6570606/15142790/fa5474f4-16e9-11e6-828a-2d618fd0241d.png)

**Comment by ElodieKongasiou** [comment]
Yep that looks fine to me :)
Happy for  a small animation as long as it doesn't take too long to do.
Katie what do you think?

x



ELODIE KONGASIOU <elodie@smackbangdesigns.com>


Account Manager

p.   <0413+072+556>02 9387 8198

w.  smackbangdesigns.com <http://smackbangdesigns.com/>
FOLLOW US

<https://www.facebook.com/smackbangdesigns>
<http://instagram.com/smackbangdesigns>     <http://pinterest.com/smackbang>
     <http://www.theloop.com.au/smackbangdesigns1/overview>

PLEASE THINK OF THE TREES BEFORE YOU PRINT THIS EMAIL.

**Comment by chausme** [comment]
Good point Mark, to make website pages and code consistent we can use animation from Furniture pages, eg. http://smackbangdevelopment.com/tmbrmll/what-we-do/furniture/
